src/components/Tooltip/index.tsx

```diff
@@ -39,13 +39,24 @@ export default function Tooltip({ text, width, size, ...rest }: TooltipProps) {
   )
 }
 
-export function MouseoverTooltip({ children, ...rest }: Omit<TooltipProps, 'show'>) {
+export function MouseoverTooltip({
+  children,
+  pointerEvent,
+  ...rest
+}: Omit<TooltipProps, 'show'> & { pointerEvent?: boolean }) {
+  // pointerEvent: when use MouseoverTooltip together with disabled button - onMouseEnter onMouseLeave not work correctly, use pointer event will fixed that issue
```

Comment:
change props name will be work. dont need onPointerEvent here. @namgold and @hungdoansy for more detail